### PR TITLE
Use a rough init script to run the demo site as a service at startup

### PR DESCRIPTION
**Pull request by paulerickson**

### CODE REVIEW

**Comment by nblair** [comment]
This is rad. How tough would it be to get the admin side running similarly? Part of he target audience that will be using this instance during evaluations will need to see that side.

File: broadleaf-demo.initscript [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  start)
+        log_daemon_msg "Starting $DESC." "$NAME"
+        if [ ! -d "/home/$JETTY_USER/.m2/repository" ]; then
+                log_daemon_msg "Initializing maven repository."
+                su $JETTY_USER -c "cd $PROJECTDIR; mvn install"
+        fi
+	# running under bash -c exec gives the right pid and allows output to be captured
+        start-stop-daemon --start --pidfile "$PIDFILE" --chuid "$JETTY_USER" \
+                --chdir "$JETTY_HOME" --background --make-pidfile \
+                --startas /bin/bash -- -c "exec /usr/bin/ant jetty-demo > stdout.log 2>&1"
```
Comment (nblair) [inline broadleaf-demo.initscript:41]: :+1: on technique, I like that the log is available.

Comment (paulerickson) [inline broadleaf-demo.initscript:41]: Oh right, the admin service is next, and it ought to be just more of the same.

**Comment by nblair** [comment]
Go ahead and add it to this PR then if you can, I should have mentioned it explicitly in the JIRA task.

Thanks, this is awesome!

**Comment by paulerickson** [comment]
Okay, the admin service is in now too.

TIL that Windows line-endings can break shell scripts.  Not sure how that flipped on me, but I'm pretty sure that git never pushed the bad CRLF line endings.

**Comment by nblair** [comment]
Check with Andrew Hoffman, I think there is a git config option to avoid issues with CRLF.

This is awesome, merging!

**Pull request merged**